pkg/api/soc.go

```diff
@@ -53,13 +67,21 @@ func (s *server) socSigUploadHandler(w http.ResponseWriter, r *http.Request) {
 		jsonhttp.InternalServerError(w, "cannot read chunk data")
 		return
 	}
+
 	if len(data) < swarm.SpanSize {
 		s.Logger.Debugf("soc upload: chunk data too short")
-		s.Logger.Error("soc upload: chunk data")
+		s.Logger.Error("soc upload: %v", errBadRequestParams)
 		jsonhttp.BadRequest(w, "short chunk data")
 		return
 	}
 
+	if len(data) > swarm.ChunkSize {
```

Comment:
wait, now we said Spanbytes is part  of the  data, then you need to use a different constant and amend comment 4096 --> 4104